src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,140 @@
+package duke;
+

```

Comment:
I noticed that there's always 2 blank lines between the package and the imports, perhaps you can set your IDE settings to make it 1, so you do not have to worry about this in the future.